algopy_testing/src/algopy_testing/constants.py

```diff
@@ -1,6 +1,8 @@
 UINT64_SIZE = 64
 UINT512_SIZE = 512
 MAX_BYTES_SIZE = 4096
+MAX_STRING_SIZE = 4096
```

Comment:
I don't think we want a seperate size for String, as they have the same limitation as the bytes type in terms of size, it's just the Stubs have them as a distinct type